config/jobs/kubernetes/sig-testing/bazel-build-test.yaml

```diff
@@ -29,6 +29,34 @@ presubmits:
           requests:
             memory: "6Gi"
 
+  - name: pull-kubernetes-bazel-build-rbe
+    skip_report: true
+    always_run: true
+    skip_branches:
+    - release-1.14 # per-release job
+    - release-1.13 # per-release job
+    - release-1.12 # per-release job
+    - release-1.11 # per-release job
+    labels:
+      preset-service-account: "true"
+      preset-bazel-scratch-dir: "true"
+    extra_refs:
+    - base_ref: master
+      org: kubernetes
+      repo: test-infra
+    spec:
+      containers:
+      - image: launcher.gcr.io/google/bazel:0.24.1
+        command:
+        - ../test-infra/hack/bazel.sh
+        args:
+        - build
+        - --config=remote
+        - //...
+        - //build/release-tars
+        - --
```

Comment:
nit: Is this necessary?

Reply:
Yes otherwise it gives an error that `-//vendor/...` is an unknown flag